test-sites/ocw-ci-test-course/content/resources/ocw_test_course_mit8_01f16_l01v01_360p.md

```diff
@@ -11,8 +11,8 @@ image_metadata:
 learning_resource_types: []
 license: https://creativecommons.org/licenses/by-nc-sa/4.0/
 ocw_type: ""
-optional_tab_title: ""
-optional_text: ""
+optional_tab_title: "Optional Tab"
+optional_text: "[OCW](https://ocw.mit.edu)"
 parent_title: Test Course Video
```

Comment:
@gumaerc Should this video actually be showing a transcript tab?

The transcript is `""`. I thought hugo treated that like `null` ?

Reply:
This is a good point. On line 26 of `video.html`, we have:

`{{ if $captionsLocation }}`

First of all, this is only checking the captions location. I assume this is because the "Transcript" tab actually uses the captions file with VideoJS to render the interactive transcript, then the PDF is offered with a download button. Furthermore, both of the raw values in `.Params` were being passed through `resource_url.html` before checking them, which has the effect of prefixing them with `RESOURCE_BASE_URL` when testing locally, which makes them not a blank string. My updated logic checks the raw params to form a boolean about whether or not the expandable tab should be rendered and adds a test to assert that it works the way it should.